### ISSUE TITLE
Very nice!

### ISSUE THREAD

**Issue by andymans** [vygr/OSX-Forth#2]
Chris,

Nice work - very cool. I'm hoping to find time to get it working on Catalina with 64-bit.

**Comment by vygr** [comment]
Many thanks for the nice words ! :)

You may also like to see the OS project I started after I did the Forth project, took a look at Lisp this time too.

https://github.com/vygr/ChrysaLisp

Chris

**Comment by andymans** [comment]
Yes - ChrysaLisp looks very interesting. I'll try and grab it on the weekend and have a play around when I've got a few contiguous hours to look at it properly :-)

**Comment by deadmarshal** [comment]
i386 is deprecated on new MacOSes. Probably need to update this to 64 bit.

**Comment by andymans** [comment]
I think that's left as an exercise for the reader, @deadmarshal - but Chris has provided a great example of how to build a "pro" Forth here - only needs re-working to use the 64-bit registers etc.  That said...Apple's docs for their assembler are notoriously hard to pin down...

**Comment by vygr** [comment]
Apples code execution privilege changes they did a while back would probably stop this working now.

Due to how it organises the dictionary. It has executable code mixed in with fourth word headers.

I keep promising myself that I might get back to doing some Forth at some point. But I'm afraid doing my own dialect of Lisp and its associated OS is consuming all my attention at the moment.
